src/main/java/com/github/vkorobkov/jfixturescmd/CmdArgs.java

```diff
@@ -0,0 +1,94 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.github.lalyos.jfiglet.FigletFont;
+import com.github.vkorobkov.jfixtures.JFixtures;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResultImpl;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.cli.*;
+
+import java.lang.reflect.Method;
+
+import static com.github.vkorobkov.jfixturescmd.Prompt.getProperties;
+
+@Slf4j
+class CmdArgs {
+    private String[] args = null;
+    private Options options = new Options();
+
+    CmdArgs(String[] args) {
+        this.args = args;
+        this.options.addOption("y", "srcYaml", true, "set source YAML folder path");
+        this.options.addOption("s", "outputSql", true, "set output SQL file path");
+        this.options.addOption("t", "sqlType", true, "set SQL type: " + SqlTypes.getValues().toLowerCase());
+        this.options.addOption("h", "help", false, "print help message");
+        this.options.addOption("verbose", "be extra verbose");
+        this.options.addOption("version", "print the version information and exit");
+        this.options.addOption("console", "print result SQL to console");
+    }
+
+    @SneakyThrows
+    void parse() {
+        try {
+            if (args.length == 0) {
+                help();
+            } else {
+                CommandLineParser parser = new DefaultParser();
+                CommandLine commandLine = parser.parse(options, args);
+
+                if (commandLine.hasOption("h")) {
+                    help();
+                } else if (commandLine.hasOption("version")) {
+                    version();
+                } else {
+                    parseCmdArguments(commandLine);
+                }
+            }
+        } catch (ParseException e) {
+            log.error("Failed to parse command line arguments: " + e.getMessage());
+            help();
+        }
+    }
+
+    @SneakyThrows
+    private void parseCmdArguments(final CommandLine commandLine) throws ParseException {
```

Comment:
Any chance to make command line parser be checking the required parameters? It looks like a lot of routine if-else cases which we possibly can avoid